mobile-widgets/qmlmanager.cpp

```diff
@@ -154,7 +154,8 @@ QMLManager::QMLManager() : m_locationServiceEnabled(false),
 	m_selectedDiveTimestamp(0),
 	alreadySaving(false),
 	m_pluggedInDeviceName(""),
-	m_showNonDiveComputers(false)
+	m_showNonDiveComputers(false),
+	m_oldStatus(qPrefCloudStorage::CS_UNKNOWN)
```

Comment:
the commit message makes no sense. 
`Removal of oldStatus in qmlmanager proved to be complicated,
so instead the variable is moved into qmlmanager.`
Can you rephrase this?

Reply:
refrased.